### DISCUSSION TITLE
Slight Customizations on ConnectButton Modal

### DISCUSSION THREAD

**Discussion by mubarakone** [rainbow-me/rainbowkit#1502] [Q&A]
I understand the best way to fully customize your ConnectButton component is to use ConnectButton.Custom. However, I only want to _slightly_ change the component, which means I will be needing all the original styles of ConnectButton to apply for ConnectButton.Custom. 

![image](https://github.com/rainbow-me/rainbowkit/assets/52806204/cd9e81f7-25f0-4a65-b5bc-9dcfaefb5523)
`Circled in red, I will simply be changing the contents inside the circle but keeping everything else the same`

How would I go about this process? Is there an easier way that I haven't realized?

**Comment by 0xmkh** [comment]
@mubarakone There isn't a simple process to change the content inside a modal in fact it's not possible. We would have to create a new feature or expose a hook that will allow you to edit those contents.
